pytripgui/app_logic/app_callbacks.py

```diff
@@ -309,6 +309,42 @@ def export_patient_dicom_callback(self, patient_item):
         logger.debug("DICOM export finished.")
         return True
 
+    def export_dose_voxelplan_callback(self, result_item):
+        # TODO docstring
+        logger.debug("Export DoseCube to Voxelplan")
+        dose_cube = result_item.data
+        # patient = result_item.parent.data
+        full_path = self.parent_gui.save_file_path("Export Dose to Voxelplan", "Voxelplan (*.hed)")
+
+        if not full_path:
```

Comment:
add comment why are you returning here from that method

Reply:
Comment added.